feature/eks-cluster-creation/cluster-metrics-server.tf

```diff
@@ -0,0 +1,4 @@
+module "metrics_server" {
```

Comment:
why we need it?

Reply:
Yes it is, as cluster autoscaler took metrics about current cluster usage, which are created by metrcis-server. Without that, cluster-autoscaler wont work.